examples/cozmo_face_mirror.py

```diff
@@ -69,17 +69,17 @@ def cozmo_face_mirror(robot):
         latest_image = robot.world.latest_image
 
         if latest_image is not None:
-            # scale the camera image down to fit on Cozmo's face
+            # Scale the camera image down to fit on Cozmo's face
             resized_image = latest_image.raw_image.resize(face_dimensions,
                                                           Image.BICUBIC)
 
             # Flip the image left/right so it displays mirrored
             resized_image = resized_image.transpose(Image.FLIP_LEFT_RIGHT)
 
-            # calculate the pixel threshold for this image
+            # Calculate the pixel threshold for this image
             pixel_threshold = calc_pixel_threshold(resized_image)
 
-            # convert the image to the format to display on Cozmo's face
+            # Convert the image to the format to display on Cozmo's face
```

Comment:
Each of these two lines ("Convert the..." and "the pixel_threshold") should start with a capital and end with a period to make it more readable.